modules/ip6/cli/route.c

```diff
@@ -74,11 +73,9 @@ static cmd_status_t route6_list(const struct gr_api_client *c, const struct ec_p
 	for (size_t i = 0; i < resp->n_routes; i++) {
 		struct libscols_line *line = scols_table_new_line(table, NULL);
 		const struct gr_ip6_route *route = &resp->routes[i];
-		ip6_net_format(&route->dest, dest, sizeof(dest));
-		inet_ntop(AF_INET6, &route->nh, nh, sizeof(nh));
 		scols_line_sprintf(line, 0, "%u", route->vrf_id);
-		scols_line_set_data(line, 1, dest);
-		scols_line_set_data(line, 2, nh);
+		scols_line_sprintf(line, 1, "%6p/%hhu", &route->dest, route->dest.prefixlen);
+		scols_line_sprintf(line, 2, "%6p", &route->nh);
```

Comment:
typical complexity here, FMT is needed for "%6p"

Reply:
done